multiplayer/src/components/ReactionParticle.tsx

```diff
@@ -25,12 +29,25 @@ export default function Render(props: { particle: Particle }) {
         };
     }, []);
 
+    const override = reactionIconOverrides?.[index + 1];
```

Comment:
Why do you need the `+ 1`?

Reply:
reactions are 0 indexed but displayed / used as 1 index by users (see `/Reaction.tsx`); `reactionIconOverrides` come directly from user code so I left the indexing there as 1 indexed there to make it a bit easier to match state vs user action (and also to be consistent with the player slots which are 1 indexed), so need to